plugins/00_dokku-standard/subcommands/run

```diff
@@ -23,6 +24,16 @@ dokku_run_cmd() {
   [[ "$DOKKU_RM_CONTAINER" ]] && local DOKKU_RUN_OPTS="--rm"
   has_tty && local DOKKU_RUN_OPTS+=" -i -t"
   is_image_herokuish_based "$IMAGE" && local EXEC_CMD="/exec"
+
+  # TODO: should we cleanup the procfile after run?
+  extract_procfile "$APP"
+  PROC_CMD=$(get_cmd_from_procfile "$APP" "$1" || echo '')
+
+  if [ ! -z $PROC_CMD ]; then
+    dokku_log_info1 "Found '$1' in Procfile, running that command"
```

Comment:
Is `dokku_log_info1` the correct log function to use for these kinds of messages?